src/main/java/org/openjdk/jextract/json/parser/JWCC.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright (c) 2024, Oracle and/or its affiliates. All rights reserved.
+ * DO NOT ALTER OR REMOVE COPYRIGHT NOTICES OR THIS FILE HEADER.
+ *
+ * This code is free software; you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License version 2 only, as
+ * published by the Free Software Foundation.
+ *
+ * This code is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License
+ * version 2 for more details (a copy is included in the LICENSE file that
+ * accompanied this code).
+ *
+ * You should have received a copy of the GNU General Public License version
+ * 2 along with this work; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin St, Fifth Floor, Boston, MA 02110-1301 USA.
+ *
+ * Please contact Oracle, 500 Oracle Parkway, Redwood Shores, CA 94065 USA
+ * or visit www.oracle.com if you need additional information or have any
+ * questions.
+ */
+package org.openjdk.jextract.json.parser;
+
+/**
+ * JWCC is JSON With Commas and Comments. In addition to supporting all of JSON
+ * JWCC also supports trailing commas and comments. Comments can be either
+ * until single-line or multi-line.
+ *
+ * Comments are stripped and are not present in the parsed result.
+ */
+public final class JWCC {
```

Comment:
This class (and related test) don't seem very useful. I think we can just drop them

Reply:
That means we'll not support extensions like comments and trailing comma (optionally). I'll remove that JSON.parse method then.